x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/ilm/ForceMergeActionTests.java

```diff
@@ -33,36 +35,28 @@ protected ForceMergeAction createTestInstance() {
     }
 
     static ForceMergeAction randomInstance() {
-        return new ForceMergeAction(randomIntBetween(1, 100));
+        return new ForceMergeAction(randomIntBetween(1, 100), randomBoolean());
     }
 
     @Override
     protected ForceMergeAction mutateInstance(ForceMergeAction instance) {
         int maxNumSegments = instance.getMaxNumSegments();
-        maxNumSegments = maxNumSegments + randomIntBetween(1, 10);
-        return new ForceMergeAction(maxNumSegments);
+        boolean bestCompression = instance.isBestCompression();
+        if(randomBoolean()) {
```

Comment:
Another super minor thing
```suggestion
        if (randomBoolean()) {
```